backend-rust/src/router.rs

```diff
@@ -8,10 +10,14 @@ use tokio_util::sync::CancellationToken;
 pub async fn serve(
     registry: Registry,
     tcp_listener: TcpListener,
+    pool: PgPool,
     stop_signal: CancellationToken,
 ) -> anyhow::Result<()> {
-    let app =
-        axum::Router::new().route("/", axum::routing::get(metrics)).with_state(Arc::new(registry));
+    let health_routes = Router::new().route("/", get(health)).with_state(pool);
+
+    let metric_routes = Router::new().route("/", get(metrics)).with_state(Arc::new(registry));
+
+    let app = Router::new().nest("/metrics", metric_routes).nest("/health", health_routes);
```

Comment:
Small thing, but currently the command line argument/ environment variable for this service is name to only fix for metrics, maybe rename the [argument](https://github.com/Concordium/concordium-scan/blob/main/backend-rust/src/bin/ccdscan-api.rs#L36) (both `ccdscan-api.rs` and `ccdscan-indexer.rs`).

Reply:
I generally renamed all related fields to monitoring